### PR TITLE
Z index

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

File: scripts/visualize/visualize_utils.R [hunk 1]
```diff
@@ -230,7 +230,7 @@ createHurricaneSnapshot <- function(fig_height, fig_width, css, time_stamp, stat
   state_lwd <- locate_css_class_detail(state_css, "stroke-width")
   island_css <- locate_css_class(css, ".island-polygon")
   island_color <- locate_css_class_detail(island_css, "fill")
-  island_lwd <- locate_css_class_detail(island_css, "stroke-width")
+  island_lwd <- 0
```
Comment (wdwatkins) [inline scripts/visualize/visualize_utils.R:233]: `island_lwd` isn't used anywhere else

**Comment by jordansread** [comment]
image

**Comment by amrhoades** [comment]
Looking good! What if the state line was one or two shades lighter?

**Comment by jordansread** [comment]
I made the borders lighter

**Pull request merged**